src/precomputed-tables.lisp

```diff
@@ -41,7 +41,7 @@
 
 
 (defparameter *composition-exclusions-data* (make-hash-table))
-(with-open-file (in (uiop:merge-pathnames* *data-directory* "CompositionExclusions.txt"))
+(with-open-file (in (uiop:merge-pathnames* *data-directory* "CompositionExclusions.txt" :external-format :UTF-8))
```

Comment:
This is a bad call to `UIOP:MERGE-PATHNAMES*` - the parens are misplaced, the `:external-format` argument should be part of `WITH-OPEN-FILE`. Same issue below.

Reply:
Thanks. Will fix.